AppFramework/Matcher/GREYMatchers.m

```diff
@@ -151,7 +151,11 @@ + (void)initialize {
     if (element.accessibilityIdentifier == accessibilityID) {
       return YES;
     }
-    return [element.accessibilityIdentifier isEqualToString:accessibilityID];
+
```

Comment:
how about `[accessibilityID isEqualToString:element.accessibilityIdentifier]`?

Reply:
This is actually a really good idea.

Reply:
This is actually a really good idea.